app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -100,14 +100,14 @@ const SearchResult = ({ hit, index, setCenterCoords }) => {
   const clinicianActionsLink = (
     <div className={styles.sideLink} role="button" tabIndex={0} onClick={toggleClinicianActionsModal}>
       <img src={icon('clinician-action')} alt="clinician action" className={styles.sideLinkIcon} />
-      <div className={styles.sideLinkText}>Clinician action</div>
+      <div className={styles.sideLinkText}>Clinician actions</div>
     </div>
   );
 
   const handoutsLink = (
     <div className={styles.sideLink} role="button" tabIndex={0} onClick={toggleHandoutModal}>
       <img src={icon('print-blue')} alt="printout icon" className={styles.sideLinkIcon} />
-      <div className={styles.sideLinkText}>Client handouts</div>
+      <div className={styles.sideLinkText}>Print</div>
```

Comment:
This comment is probably more about the design than the code, but this new label is actually a little bit confusing to me as a user. When I click on it, instead of printing, it displays a model with the title "Handouts" with links to PDFs.

Maybe the main thing that is confusing to me is the title of the modal, since when it initially opened, it made me think that I clicked on the wrong thing because it doesn't match the label on the button.

Reply:
Yes, I found that confusing as well. 

I also wondered if the Print/Handout link and the Clinician Action link should be one after another so the user understands that they are part of the same "user flow", for lack of a better term. Thanks for the suggestion! I have updated it.